src/v1/account_validity.rs

```diff
@@ -0,0 +1,59 @@
+//! [GET /_synapse/admin/v1/account_validity/validity](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/account_validity.rst)
+
+use std::time::SystemTime;
+
+use ruma::api::ruma_api;
+use ruma::identifiers::UserId;
+
+ruma_api! {
+    metadata: {
+        description: "Extend the account validity.",
+        method: POST,
+        name: "set_account_validity",
+        path: "/_synapse/admin/v1/account_validity/validity",
+        rate_limited: false,
+        authentication: None, // AccessToken?
+    }
+
+    request: {
+        /// user ID for the account to renew
+        pub user_id: &'a UserId,
+
+        /// This is an optional parameter, it overrides the expiration date,
+        /// which otherwise defaults to now + validity period configured at the server
+        #[serde(
+            with = "ruma::serde::time::opt_ms_since_unix_epoch",
+            skip_serializing_if = "Option::is_none"
+        )]
+        pub expiration_ts: Option<SystemTime>,
+
+        /// This is an optional parameter, it enables/disables sending renewal emails to the user.
+        /// Defaults to true.
+        #[serde(default = "ruma::serde::default_true")]
```

Comment:
If it has a default attribute, it should usually also have `skip_serializing_if`(`ruma::serde::is_true`).

Reply:
done